src/examples/table/TableDraggable.tsx

```diff
@@ -86,11 +87,23 @@ const SortableRow = ({ item }: { item: IItem }) => {
       ref={setNodeRef}
       style={{
         transform: CSS.Transform.toString(transform),
-        transition
+        transition,
+        backgroundColor: isDragging
+          ? getColor({
+              variable: 'background.primaryEmphasis',
+              transparency: theme.opacity[300],
+              dark: { offset: -100 },
+              theme
+            })
+          : undefined
       }}
     >
       <Table.Cell>
-        <DraggableContainer {...attributes} {...listeners}>
+        <DraggableContainer
+          {...attributes}
+          {...listeners}
+          style={isDragging ? { cursor: 'grabbing' } : undefined}
```

Comment:
There seems to be a problem where the `grabbing` cursor reverts to default on drag. Can this be fixed?


https://github.com/user-attachments/assets/70ada191-91f7-47c1-88e8-6c19cede50ca

Reply:
After hours of troubleshooting, I still wasn't able to pinpoint the cause of the weird cursor behavior. However, I did find a solution that improves the UX. Now, the cursor occasionally switches from `grabbing` to `grab`.

https://github.com/user-attachments/assets/76a08645-477c-4e2a-9123-0b1f1f2768e7

I hope this is an acceptable compromise.

Reply:
Sounds like a `dnd-kit` problem – probably not worth pouring more time into. LGTM

Reply:
Very weird. I dont think i've ever seen this problem... wonder if it exists in the dnd kit examples for draggable components.